.idea/misc.xml

```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should add this file back, and it's contents should be exactly as in: https://github.com/unifyai/ivy/blob/master/.idea/misc.xml